packages/lesswrong/lib/collections/comments/views.ts

```diff
@@ -298,10 +298,19 @@ Comments.addView("postLWComments", (terms: CommentsViewTerms) => {
   };
 })
 
-Comments.addView("profileRecentComments", (terms: CommentsViewTerms) => {
+export const profileCommentsSortings: Partial<Record<CommentSortingMode,MongoSelector<DbComment>>> = {
+  "new" :  { isPinnedOnProfile: -1, postedAt: -1},
+  "top" : { baseScore: -1},
+  "old": {postedAt: 1},
+  "recentComments": { lastSubthreadActivity: -1 },
+} as const;
+
+Comments.addView("profileComments", (terms: CommentsViewTerms) => {
```

Comment:
Just as an FYI, this is a breaking change to external users of our API - i.e. this might break parts of GreaterWrong if they're using this view.  Instead of changing the existing one I would make a new one and leave a comment above the existing one indicating that it's deprecated with a pointer to the new one.

Reply:
Okay, I will keep the old view and leave a comment flagging it as deprecated.